homework/12_B/19_Kristian/src/main/java/com/kris/circles/algorithms/CircleAlgorithms.java

```diff
@@ -0,0 +1,45 @@
+package com.kris.circles.algorithms;
+
+import com.kris.circles.Circle;
+
+public class CircleAlgorithms {
+  private Circle firstCircle;
+  private Circle secondCircle;
+
+  public CircleAlgorithms(Circle one, Circle two) {
+    firstCircle = one;
+    secondCircle = two;
+  }
+
+  public void run() {
+    System.out.println(calculate());
+  }
+
+  private String calculate() {
+    double distance = Math.sqrt(Math.pow(Math.abs(firstCircle.getX() - Math.abs(secondCircle.getX())), 2)
+        + Math.pow(Math.abs(firstCircle.getY() - Math.abs(secondCircle.getY())), 2));
```

Comment:
That expression is overly complicated. I suggest you break it down into a few smaller expressions.

Also as far as I can see you are calculating:

$$ distance = \sqrt{|c1.x - |c2.x||^2 + |c2.y - |c2.y||^2} $$

Which is not the right formula to compute the distance between two points!